h/groups/schemas.py

```diff
@@ -23,7 +23,7 @@ class GroupSchema(CSRFSchema):
             min=GROUP_NAME_MIN_LENGTH,
             max=GROUP_NAME_MAX_LENGTH),
         widget=deform.widget.TextInputWidget(
-            autofocus=True,
```

Comment:
I'm a bit confused by this. You didn't disable the focus outline, you just disabled autofocus...